src/auto-evo/RunResults.cs

```diff
@@ -205,6 +221,7 @@ void OutputPopulationForPatch(Species species, Patch patch, long population)
 
                 if (entry.SpreadToPatches.Count > 0)
                 {
+                    //totalMovedPopulation += entry.SpreadToPatches.Count;
```

Comment:
remove this comment

Reply:
Anything else?

Reply:
?
https://github.com/Revolutionary-Games/Thrive/pull/1692#discussion_r503225013
That needs logic adjustment.